src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -0,0 +1,98 @@
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import classNames from 'classnames';
+
+import { changeBurgerState } from '@/features/burgermenu/burgerSlice';
+
+import fav from '../../../src/assets/images/icons/favorites-blank.svg';
```

Comment:
Dose it work this way?
```suggestion
import fav from '@/assets/images/icons/favorites-blank.svg';
```